### PR TITLE
Fabo/handle ledger in use

### PR DESCRIPTION
**Pull request by faboweb**

### CODE REVIEW

File: src/cosmos-ledger.ts [hunk 1]
```diff
@@ -101,11 +101,17 @@ export default class Ledger {
         )
         transport = await TransportWebUSB.create(timeout * 1000)
       } catch (err) {
+        /* istanbul ignore next: specific error rewrite */
         if (err.message.trim().startsWith('No WebUSB interface found for your Ledger device')) {
           throw new Error(
             "Couldn't connect to a Ledger device. Please use Ledger Live to upgrade the Ledger firmware to version 1.5.5 or later."
```
Comment (jbibla) [inline src/cosmos-ledger.ts]: this is incorrect? should say 1.5.3 now...

File: src/cosmos-ledger.ts [hunk 2]
```diff
@@ -101,11 +101,17 @@ export default class Ledger {
...
           )
         }
+        /* istanbul ignore next: specific error rewrite */
+        if (err.message.trim().startsWith('Unable to claim interface')) {
+          // apparently can't use it in several tabs in parallel
+          throw new Error('Could not access Ledger device. Do you use it in another tab?')
```
Comment (jbibla) [inline src/cosmos-ledger.ts]: `Could not access Ledger device. Is it being used in another tab?`
`Could not access Ledger device. Already in use.`

File: src/cosmos-ledger.ts (hunk 1 above)
Comment (faboweb) [reply src/cosmos-ledger.ts]: This talks about the FW not the app version

File: src/cosmos-ledger.ts (hunk 2 above)
Comment (faboweb) [reply src/cosmos-ledger.ts]: I'll put that in the next release

**Pull request merged**